hi.js

```diff
@@ -1,821 +1,821 @@
-windows.controls.kickVoteN Vote Kick [No]
-windows.controls.kpdVoteY KPD Vote [Yes]
-windows.controls.kpdVoteN KPD Vote [No]
+module.exports = `
+menu.btn.ranked रैंक का खेल
+menu.btn.shop दुकान
+menu.btn.social हब
+menu.btn.maps मैप्स
+menu.btn.mods मॉड
+menu.btn.settings सेटिंग
+menu.btn.customize रूप बदले
+menu.btn.invite इंवाइट करें
+menu.btn.invite.clicked कॉपी
+menu.btn.join जॉइन
+menu.btn.host होस्ट करें
+menu.btn.browser खेल ढूंढें
+generic.wait कृपया प्रतीक्षा करें...
+generic.loading लोड हो रहा है । ..
+generic.none कुछ भी नहीं
+generic.connecting कनेक्टिंग...
+generic.select चुनें
+generic.delete हटाएं
+generic.save सेव करें
+generic.load लोड
+generic.search खोजें
+generic.leave छोड़ें
+generic.default डिफ़ॉल्ट
+generic.submit सबमिट करें
+generic.create बनाएँ
+generic.random रैंडम
+generic.failed असफल
+generic.success सफलता
+generic.name नाम
+generic.level लेवल
+generic.score स्कोर
+generic.off बंद
+generic.defeat हार
+generic.victory विजय
+generic.matchover मैच ख़तम
+generic.matchaband मैच छोड़ दिया
+generic.follow फॉलो
+generic.unfollow अनफॉलो
+generic.followers फॉलोवर्स
+generic.following फॉलोइंग
+generic.games खेल
+generic.wins जीत
+generic.lose हार
+generic.clan क्लैन
+generic.games.played खेले गये खेल
+generic.games.won खेल जीते
+generic.games.lost खेल हारे
+generic.timep कितना समय खेला
+generic.kills हत्या
+generic.deaths मौतें
+generic.kdr KDR
+generic.time समय
+generic.logout लॉगआउट
+generic.logout.sessions सभी सत्रों से लॉगआउट
+generic.username यूजर का नाम
+generic.password पासवर्ड
+generic.register रजिस्टर
+generic.login लॉगिन
+generic.accuracy सटीकता
+player.waiting खिलाड़ियों का इंतजार कर रहे हैं ...
+player.died तुम मर गए
+player.killed द्वारा मारे गए
+player.killstreak {0} on a {1} Kill Streak
+player.inactive इनैक्टिव
+player.disconnect डिस्कनेक्ट
+timer.end अगला दौर {0} में
+leaderboard.empty खाली लॉबी
+popup.checkpoint चैकपौइंट
+app.play खेलने के लिए क्लिक करें
+app.spectate स्पैक्टेट करे
+app.play.mobile खेलने के लिए टैप करें
+app.spectate.mobile स्पैक्टेट करे
+app.play.controller START दबाएं
+app.spectate.controller स्पैक्टेट करे
+purchase.error ERROR
+purchase.error.click यहां क्लिक करें  </a> और फिर से प्रयास करें!
+purchase.error.limit आप अपनी दैनिक लेनदेन सीमा को पार कर चुकें हैं
+settings.reset रीसेट
+settings.reset.confirm क्या आप वाकई अपनी सभी सेटिंग्स रीसेट करना चाहते हैं?
+settings.import इम्पोर्ट
+settings.export ऐक्सपोर्ट
+settings.none कोई सेटिंग नहीं मिली
+settings.backup बैकअप सेटिंग्स
+settings.controls.change Keybinds
+settings.controls.press किसी भी Key को दबाएं
+settings.local.header Localization <span id="requiresRestart"><span style="color: #eb5656">*</span> requires restart</span>
+settings.local.region डिफ़ॉल्ट क्षेत्र
+settings.local.lang भाषा
+settings.network.header नेटवर्क
+settings.network.sendRate ज़्यादा Tickrate
+settings.network.netRerouting ऑटो राउटिंग (पिंग को प्रभावित करता है)
+settings.network.lagComp लैग काम्पन्सेशन
+settings.network.hitReg hitreg टैस्ट

```

Comment:
Maybe consider using "हिटरेग" instead of "hitreg". However I'm not sure since this word is kind of technical so maybe it should be left as in English.

Reply:
I think "hitreg" should be left as it is instead of translating it as "हिटरेग"

Reply:
Cool!

Reply:
I'll fix the grammar on this..